actionpack/lib/action_dispatch/middleware/templates/rescues/diagnostics.html.erb

```diff
@@ -9,7 +9,11 @@
 
 <div id="container">
   <h2><%= h @exception.message %></h2>
-
+  <%- if h @exception.message.to_s.match(/^Migrations are pending.*RAILS_ENV=development$/) %>
```

Comment:
Should we leak the exception message format in the view? And can't we match on the exception class instead of the message?

Are there other exceptions we could consider creating actions for?